src/tests/ov_helpers/ov_snippets_models/include/subgraph_mha.hpp

```diff
@@ -57,6 +57,23 @@ class MHAFunction : public SnippetsFunctionBase {
     const std::vector<ov::element::Type> precisions;
 };
 
+class MHA2DFunction : public SnippetsFunctionBase {
+public:
+    explicit MHA2DFunction(const std::vector<PartialShape>& inputShapes, const std::vector<ov::element::Type>& precisions,
+                           bool with_mul = true, bool with_reshape = true)
+        : SnippetsFunctionBase(inputShapes), with_mul(with_mul), with_reshape(with_reshape), precisions(precisions) {
+        OPENVINO_ASSERT(input_shapes.size() == 4, "Got invalid number of input shapes");
+        OPENVINO_ASSERT(precisions.size() == 4, "Got invalid number of input precisions");
+    }
+protected:
+    std::shared_ptr<ov::Model> initOriginal() const override;
+    std::shared_ptr<ov::Model> initReference() const override;
+
+    const bool with_mul = true;
+    const bool with_reshape = true;
```

Comment:
These fields are not needed, please remove them

Reply:
Removed